src/tests/ftest/dfuse/bash.py

```diff
@@ -60,52 +51,120 @@ def test_bashcmd(self):
+                output_c = os.path.join(self.dfuse.mount_dir.value, 'output_c')
+                link_name = os.path.join(self.dfuse.mount_dir.value, 'link_c')
+
+                src_f_name = os.path.join(self.dfuse.mount_dir.value, 'src.f')
+                with open(src_f_name, 'w') as fd:
+                    fd.write('      program test\n      write(*,*) "Hello"\n      end program\n')
+                output_f = os.path.join(self.dfuse.mount_dir.value, 'output_f')
+
+                src_a_name = os.path.join(self.dfuse.mount_dir.value, 'src_a.c')
+                with open(src_a_name, 'w') as fd:
+                    fd.write('#include <stdio.h>\n\nvoid fun_a(void) {\nprintf("fun_a()");\n}\n')
+                src_b_name = os.path.join(self.dfuse.mount_dir.value, 'src_b.c')
+                with open(src_b_name, 'w') as fd:
+                    fd.write('#include <stdio.h>\n\nvoid fun_b(void) {\nprintf("fun_b()");\n}\n')
+                obj_a = os.path.join(self.dfuse.mount_dir.value, 'src_a.o')
+                obj_b = os.path.join(self.dfuse.mount_dir.value, 'src_b.o')
+                lib_ab = os.path.join(self.dfuse.mount_dir.value, 'lib.a')
+                download = os.path.join(self.dfuse.mount_dir.value, 'download.html')
                 # list of commands to be executed.
-                commands = ["mkdir -p {}".format(abs_dir_path),
-                            "touch {}".format(abs_file_path1),
-                            "ls -a {}".format(abs_file_path1),
-                            "rm {}".format(abs_file_path1),
-                            "dd if=/dev/zero of={} count={} bs={}".format(
-                                abs_file_path1, dd_count, dd_blocksize),
-                            "ls -al {}".format(abs_file_path1),
-                            "filesize=$(stat -c%s '{}');\
-                            if (( filesize != {}*{} )); then exit 1;\
-                            fi".format(abs_file_path1, dd_count, dd_blocksize),
-                            "cp -r {} {}".format(abs_file_path1, abs_file_path2),
-                            "cmp --silent {} {}".format(abs_file_path1, abs_file_path2),
-                            "rm {}".format(abs_file_path2),
-                            "mv {} {}".format(abs_file_path1, abs_file_path2),
-                            "ls -al {}".format(abs_file_path2),
-                            "ls -al {}/.".format(abs_dir_path),
-                            "ls -al {}/..".format(abs_dir_path),
-                            "rm {}".format(abs_file_path2),
-                            "rmdir {}".format(abs_dir_path)]
+                commands = [f"mkdir -p {abs_dir_path}",
+                            f"touch {abs_file_path1}",
+                            f"ls -a {abs_file_path1}",
+                            f"rm {abs_file_path1}",
+                            f"dd if=/dev/zero of={abs_file_path1} count={dd_count}"
+                            f" bs={dd_blocksize}",
```

Comment:
This looks wrong, is there some odd wrapping going on here?

Reply:
This two lines were in one line. It was split into two lines since it was more than 100 characters in one line.

Reply:
I tried add 4 blank in the second line, but I got indentation issues.